config/config/flavors/prod-sc.yaml

```diff
@@ -8,7 +8,7 @@ dex:
 alerts:
   alertTo: "opsgenie"
   opsGenieHeartbeat:
-    enable: true
+    enabled: true
     name: #Fill this out
```

Comment:
Should this field actually be empty, instead of "set-me"?

Reply:
Yes if we set it to "set-me" and the operator forgot to change it, the installation would run without a problem. By leaving it empty we make sure that the user notice it since the config validation will fail unless it is set. :slightly_smiling_face: